README.md

```diff
@@ -75,9 +76,26 @@ scrape_configs:
 
 `honor_labels: true` is important because exporter returns metrics with `instance` label set.
 
+### Docker
+#### Building
+Clone repo and build it as usual docker image:
+```
+dokcer build . -t rds_exporter 
```

Comment:
**docker** build . -t rds_exporter

Reply:
Fixed in https://github.com/pdffillerdocker/rds_exporter/commit/4e36931a0807b3e836b4f63cea3352a3ee36ed68